templates/blog/search.html.twig

```diff
@@ -2,24 +2,18 @@
 
 {% block javascripts %}
     {{ parent() }}
-    {{ encore_entry_script_tags('search') }}
 {% endblock %}
```

Comment:
This javascripts block can be remove now

Reply:
Indeed, nice catch thanks!